pkg/autodiscovery/providers/kube_services.go

```diff
@@ -178,13 +186,15 @@ func (k *KubeServiceConfigProvider) parseServiceAnnotations(services []*v1.Servi
 			delete(k.configErrors, serviceID)
 		}
 
-		ignoreADTags := ignoreADTagsFromAnnotations(svc.GetAnnotations(), kubeServiceAnnotationPrefix)
-
+		ignoreAdForHybridScenariosTags, _ := strconv.ParseBool(svc.GetAnnotations()[kubeServiceAnnotationPrefix+"ignore_autodiscovery_tags"])
```

Comment:
maybe we can have a function for clarity 

```golang
func ignoreADTagsFromAnnotations(annotations map[string]string, prefix string) bool {
  ignore := false
  if val, found := svc.GetAnnotations()[prefix+"ignore_autodiscovery_tags"]; found {
    ignore, _ = strconv.ParseBool(val)
  }
  return ignore
}

```suggestion
ignoreAdForHybridScenariosTags := ignoreADTagsFromAnnotations(svc.GetAnnotations(),kubeServiceAnnotationPrefix)
```